packages/core/src/helpers/space/space.ts

```diff
@@ -10,16 +10,17 @@
  * // '32px'
  */
 export function space(multiplier: number): string {
-  if (!isAllowed(multiplier)) {
-    const msg = '"multiplier" must be an integer or (1.5, 0.5, 0.25, 0.125)';
-    throw new Error(`${msg}. Got: ${multiplier}`);
-  }
+  if (!isAllowed(multiplier))
+    throw new Error(
+      `"multiplier" must be an integer or (${floats}). Got: ${multiplier}`
+    );
```

Comment:
I like that allowed fractionals are reused, however not sure `floats` constant is needed - won't be reused anywhere. So what if:

```suggestion
    throw new Error(
      `"multiplier" must be an integer or (${[...allowedFractionals].join(', ')}). Got: ${multiplier}`
    );
```

Reply:
That was my first approach, the problem is prettier 🙊 
```
throw new Error(
    `"multiplier" must be an integer or (${[...allowedFractionals].join(
      ', '
  )}). Got: ${multiplier}`
);
```

Reply:
Not sure, but I would like this to match Sass function as close as possible.

```ts
export function space(multiplier: number): string {
  if (!isAllowed(multiplier))
    throw new Error(
      `"multiplier" must be an integer or one of (${[
        ...allowedFractionals,
      ].join(', ')}). Got: ${multiplier}`
    );

  return `${multiplier * base}px`;
}
```

I also think that Sass function should also use `multiplier`, so:

```scss
@function space($multiplier) {
  @if $multiplier % 1 != 0 and not index($allowed-fractionals, $multiplier) {
    @error '$multiplier must be an integer or one of (#{$allowed-fractionals})';
  }

  @return $multiplier * $base;
}
```

Reply:
Fair enough, take a look at the new version and let me know how that looks.